lms/static/sass/course-list.scss

```diff
@@ -6,20 +6,22 @@
  */
 
 :root {
+  
+  //new-layout
+  --primary-color:#1868B9;
+  --background-2-color:#EBF3FA;
```

Comment:
warto zmienić nazwę czego dotyczy tło np. --background-color-tooltip lub --tooltip-background-color jesli to tło tooltipu , by było wiadomo przy modyfikacji jakiego elementu dotyczy w przyszłości, pomoże to w utrzymaniu kodu.